packages/shared/src/components/sidebar/Section.tsx

```diff
@@ -24,8 +25,32 @@ interface SectionProps extends SectionCommonProps {
   title?: string;
   items: SidebarMenuItem[];
   isItemsButton: boolean;
+  isLoading?: boolean;
 }
 
+const ItemsSkeleton = () => {
```

Comment:
Created a quick loading animation for the squad section in the sidebar, as squads may not always be loaded at render time

Reply:
It's probably good to check how many squads the average normal user has to avoid shifts for the mass

Reply:
I did a query on db, looks like only ~10.000 has more then 5, so probably not bad default

Reply:
found a better query, i think

```sql
select count(*) from public.user u where (select count(1) from source_member sm where sm."userId" = u.id) > 5
```

9,009

and more then 1 is 55,435

Reply:
so probably 1 is the best 😆